pyproject.toml

```diff
@@ -30,6 +30,19 @@ license-files = ["LICENSE"]
 [tool.setuptools.packages.find]
 include = ["rapids_dask_dependency*"]
 
-[tool.ruff]
-lint.select = ["E", "F", "W", "I", "N", "UP"]
-lint.fixable = ["ALL"]
+[tool.ruff.lint]
+select = [
+    # pycodestyle (errors)
+    "E",
+    # isort
+    "I",
+    # pyflakes
+    "F",
+    # pep8-naming
+    "N",
+    # pyupgrade
+    "UP",
+    # pycodestyle (warnings)
+    "W",
```

Comment:
Love the comments here. Thanks!

Reply:
no prob! Figured I'd propose this while I'm touching this anyway. I've found comments like this really helpful for organizing this configuration, especially when it gets big like this: https://github.com/rapidsai/cudf/blob/0b32f55b1ed38507437770d21da1e4e1a1c4a17d/pyproject.toml#L30